snyk_feed.py

```diff
@@ -70,14 +73,22 @@ def fetch(self):
             res = requests.get(self.URL, headers=headers)
         except Exception as e:
             logger.error('Unable to get snyk feed. Reason: %r' % e)
```

Comment:
If we do not raise the exception, it will continue.

Reply:
probably i need to add the "return" statement after adding the error log to stop the further processing

Reply:
done